doc/src/learn/index.md

```diff
@@ -16,15 +16,12 @@ For a deep dive into Sui technology, see the [Sui Smart Contracts Platform](http
 
 The following list includes the recent updates to Sui and the documentation:
 
+* You must now specify the key scheme type as an argument (`secp256k1` or `ed25519`)  when running either the `sui keytool generate` or `sui client new-address` commands, as shown in [Adding accounts to the client](../contribute/cli-client.md#adding-accounts-to-the-client) and [Make a Move call](../build/devnet.md#make-a-move-call).
```

Comment:
i dont think "make a move call" needs this flag, just the sui cli

Reply:
> i dont think "make a move call" needs this flag, just the sui cli

Ah, I link to that Devnet doc because in that section we also include the `sui client new-address` command. I tried to make this more clear.

Still, I can remove the reference and link if you find it confusing. Just let me know. Thanks!